scripts/linters/pylint_extensions_test.py

```diff
@@ -965,26 +991,143 @@ def test_excessive_newline_above_args(self):
+
+                    Args:
+                        arg: argument. This is  description.
+
+                    Returns:
+                        int. Returns something.
+
+
+                    Yields:
+                        yield_something. Description.
+                    \"\"\"
+                    return True
+                    yield something
+        """)
+
+        message_above_yields = testutils.Message(
+            msg_id='single-space-above-yield',
+            node=node_multiple_newlines_below_returns
+        )
+
+        message_below_returns = testutils.Message(
+            msg_id='single-space-below-returns',
+            node=node_multiple_newlines_below_returns
+        )
+
+        with self.checker_test_object.assertAddsMessages(
+            message_above_yields, message_below_returns):
+            self.checker_test_object.checker.visit_functiondef(
+                node_multiple_newlines_below_returns)
+
+    def test_no_newline_below_args(self):
+        node_no_newline_below_args_end = astroid.extract_node(
+            u"""def func(arg): #@
+                \"\"\"Do something.
+
+                Args:
+                    arg: argument. Description.\"\"\"
+        """)
+
+        message_no_newline_at_end = testutils.Message(
+            msg_id='no-newline-used-at-end',
+            node=node_no_newline_below_args_end
+        )
+
+        message_below_args = testutils.Message(
+            msg_id='single-space-below-args',
+            node=node_no_newline_below_args_end
+        )
+
+        with self.checker_test_object.assertAddsMessages(
+            message_no_newline_at_end, message_below_args):
+            self.checker_test_object.checker.visit_functiondef(
+                node_no_newline_below_args_end)
+
+    def test_no_newline_below_raises(self):
+        node_no_newline_below_raises_end = astroid.extract_node(
+            u"""def func(): #@
+                    \"\"\"Raises exception.
+
+                    Raises:
+                        raises_exception. Description.\"\"\"
+                    raise exception
+        """)
+
+        message_no_newline_at_end = testutils.Message(
+            msg_id='no-newline-used-at-end',
+            node=node_no_newline_below_raises_end
+        )
+
+        message_below_raises = testutils.Message(
+            msg_id='single-space-below-raises',
+            node=node_no_newline_below_raises_end
+        )
+
+        with self.checker_test_object.assertAddsMessages(
+            message_no_newline_at_end, message_below_raises):
+            self.checker_test_object.checker.visit_functiondef(
+                node_no_newline_below_raises_end)
+
+    def test_no_newline_below_return(self):
+        node_no_newline_below_raises_end = astroid.extract_node(
+            u"""def func(): #@
+                \"\"\"Returns something.
+
+                Returns:
+                    returns_something. Description.\"\"\"
```

Comment:
Add a test with tripe quotes `\"\"\"` on a new line.

Reply:
@Hudda thanks! If you look in the above tests, I believe triple quotes on a new line is tested. [Example here](https://github.com/oppia/oppia/pull/11124/files#diff-a6c46364ff108ecb834a9ef4c00b7d77f6ed8071dceaab2d31d608c384b57023R1025-R1041). 

I understood having a newline before the triple quotes as just having a single space below the last docstring section. Please let me know if you're referring to something else.